### PR TITLE
Fix - Message Block member popup and member listing UI changes

### LINKED ISSUE #2875: Message Block member popup and member listing UI changes
**Describe the bug**

1. Load more members link in the recipient list looks bad: https://prnt.sc/1qvtxz4
2. Messages > Group Thread > Block Member Popup > we should not show blocked or suspended members in the list.

**To Reproduce**
1. Go to Message
2. Create a thread with more than 20 members
3. See error

**Screenshots**
https://prnt.sc/1qvtxz4

### PR DESCRIPTION
**Pull request by amincdev**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

Fixes #2875 

### How to test the changes in this Pull Request:

1. Create thread message with multiple users
2. Check Load more button CSS in message header
3. After that, click on Block a member 
image
4. Click on Block button
image
5. That blocked user should not be display in the popup( Block a member list in the message screen ( in 4 point screen ) ).

### Proof Screenshots or Video

https://screencast-o-matic.com/watch/crQQQgVQmKB

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Dependent on App Native
https://github.com/buddyboss/buddyboss-app-native/issues/1606

### CODE REVIEW

**Comment by shakilAshraful** [comment]
No App Native Issue is Required https://github.com/buddyboss/buddyboss-platform/issues/2875#issuecomment-915096942

**Comment by jitendrabanjara1991** [comment]
@chetansatasiya @KartikSuthar 
Please review this PR.
We have remove blocked user from block member list for messages and changes Load More design for message headers.
cc - @amincdev 
Thanks.

**Review by KartikSuthar — CHANGES_REQUESTED** [review]
@jitendrabanjara1991 
As discussed remove the object `moderated_recipients` and separate the ajax of the block member screen.

**Comment by jitendrabanjara1991** [comment]
@KartikSuthar @chetansatasiya 
Remove **moderated_recipients** object from ajax which called previously for moderated users. And now when click on **block a member** then ajax will call. Please check it video - https://screencast-o-matic.com/watch/crQojwV6eq9
Please review code.
Thanks.

**Comment by jitendrabanjara1991** [comment]
@KartikSuthar @chetansatasiya
I have some changes in code - like remove unwanted action from js also remove current user from recipients list for message header recipients. Now you can review code.
Thanks.

File: src/bp-templates/bp-nouveau/includes/messages/ajax.php [hunk 1]
```diff
@@ -2680,3 +2683,129 @@ function bb_nouveau_ajax_recipient_list_for_blocks() {
 		)
 	);
 }
+
+/**
+ * Function which get moderated recepient list when click on block a member in the message screen.
+ */
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/includes/messages/ajax.php:2689]: @jitendrabanjara1991 Add all the detail in dock block

**Review by chetansatasiya — APPROVED** [review]
@jitendrabanjara1991 LGTM only one change is to add proper function doc block with since tag other details.

File: src/bp-templates/bp-nouveau/includes/messages/ajax.php (hunk 1 above)
Comment (jitendrabanjara1991) [reply src/bp-templates/bp-nouveau/includes/messages/ajax.php]: @chetansatasiya Added since tag and return type there

**Comment by jitendrabanjara1991** [comment]
@KartikSuthar @chetansatasiya 
Added loader when click on block a member button. So Modified some code like - some html. So, Please review code again.
Here is the video in which I have tested after above all changes - https://screencast-o-matic.com/watch/crQD22V6jVZ
cc - @amincdev 
Thanks

**Review by chetansatasiya — APPROVED** [review]

**Comment by jitendrabanjara1991** [comment]
Ready for testing.
Merged in dev and qa-ongoing.
Thanks.

**Comment by sumitbuddyboss** [comment]
Hello @jitendrabanjara1991 @amincdev 

**Issue:**

If there is only 3 member in the message thread although "Load more" button appears.
https://www.loom.com/share/6f7f3b294ff14e59aee84d726c632102

Let me know if you need any help to reproduce the issue.

Thank you

**Comment by sumitbuddyboss** [comment]
Hello @jitendrabanjara1991 @amincdev 

**Fixed >**

A related issue is resolved, now block and suspend user not display in the popup and "Load more" button will appear if we have 20+users, for the proof please check the below video.

**Video:**
https://www.loom.com/share/e981bae043d242b8969365f2307ba497

Ready for release 🚀 

Thank you

**Pull request merged**